.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  forge:
+  tests:
     name: Run Unit and E2E Tests
```

Comment:
Let's separate the unit and integration tests, they should not depend on one another